Update Dockerfile and doc (#1999)
pytorch/fairseq: examples/simultaneous_translation/docs/evaluation.md

For text, the unit is detokenized token.
For speech, the unit is millisecond.

## Final Evaluation with Docker
Our final evaluation will be run inside Docker. To run evaluation with Docker, first build a Docker image from the Dockerfile. Here is an [example](../Dockerfile) 
```bash
docker build -t iwslt2020_simulast:latest .
```
When submitting your final models, define a client command that would run inside the docker in a Dockerfile. For example, to evaluate the text translation in [baseline](baseline.md) experiment, the model can be evaluated as follow.

```bash
CLIENT_COMMAND="./examples/simultaneous_translation/scripts/start-multi-client.sh ./examples/simultaneous_translation/scripts/configs/must-c-en_de-text-dev.sh experiments/checkpoints/checkpoint_text_waitk3.pt"

docker run --env CLIENT_COMMAND=$CLIENT_COMMAND -v "$(pwd)"/experiments:/fairseq/experiments -it iwslt2020_simulast
```

> `CLIENT_COMMAND` can be the client command for customized client.

Reviewer: ```suggestion
`CLIENT_COMMAND` can be the client command for a customized client.
```

> When submitting you docker file, please keep the server settings in [example](../Dockerfile) and make sure it works for dev and open test set. We will modify the docker file for the blind test set.

Reviewer: ```suggestion
When submitting you Docker file, please keep the server settings in [example](../Dockerfile) and make sure it works for dev and open test set. During the official eveluation, we will run the Docker file with different environment variables corresponding to the blind test sets.
```

For text, the unit is detokenized token.
For speech, the unit is millisecond.

## Final Evaluation with Docker
Our final evaluation will be run inside Docker. To run an evaluation with Docker, first build a Docker image from the Dockerfile. Here is an [example](../Dockerfile) 
```bash
docker build -t iwslt2020_simulast:latest .
```
When submitting your final models, define a client command that would run inside the docker in a Dockerfile. For example, to evaluate the text translation in [baseline](baseline.md) experiment, the model can be evaluated as follow.

```bash

> CLIENT_COMMAND="./examples/simultaneous_translation/scripts/start-multi-client.sh ./examples/simultaneous_translation/scripts/configs/must-c-en_de-text-dev.sh experiments/checkpoints/checkpoint_text_waitk3.pt"

Reviewer: ```suggestion
CLIENT_COMMAND="./examples/simultaneous_translation/scripts/start-multi-client.sh ./examples/simultaneous_translation/scripts/configs/must-c-en_de-text-dev.sh experiments/checkpoints/checkpoint_text_waitk3.pt"
SRC_FILE=experiments/data/must_c_1_0/en-de/bi-text/tst-COMMON.en
TGT_FILE=experiments/data/must_c_1_0/en-de/bi-text/tst-COMMON.de
PORT=12321
```

> docker run --env CLIENT_COMMAND=$CLIENT_COMMAND -v "$(pwd)"/experiments:/fairseq/experiments -it iwslt2020_simulast

Reviewer: ```suggestion
docker run --env CLIENT_COMMAND="$CLIENT_COMMAND" -e SRC_FILE="$SRC_FILE" -e TGT_FILE="$TGT_FILE" -e PORT="$PORT" -v "$(pwd)"/experiments:/fairseq/experiments -it iwslt2020_simulast
```
